presentation/src/main/java/com/pocs/presentation/view/post/edit/PostEditScreen.kt

```diff
@@ -27,12 +27,13 @@ import com.pocs.presentation.view.component.textfield.SimpleTextField
 import kotlinx.coroutines.launch
 
 @Composable
-fun PostEditScreen(uiState: PostEditUiState, navigateUp: () -> Unit, onSuccessSave: () -> Unit) {
+fun PostEditScreen(uiState: PostEditUiState, navigateUp: () -> Unit, onSuccessSave: () -> Unit, chipsVisible: Boolean) {
```

Comment:
이렇게 매개변수로 전달하기보다는 PostEditUiState에 `showChips`라는 속성을 추가하는게 좋아보입니다.
```suggestion
fun PostEditScreen(uiState: PostEditUiState, navigateUp: () -> Unit, onSuccessSave: () -> Unit) {
```